### PR TITLE
Prod env

### PR DESCRIPTION
**Pull request by shapi78**
- Generate subdomain per environment
- Calculates the VPC subnet to pass to VPN

### CODE REVIEW

File: lib/facter/ec2_tag_facts.rb [hunk 1]
```diff
@@ -105,6 +124,22 @@ def debug_msg(txt)
 
           # append to the hash for structured fact later
           result[name] = tag['value']
+          if name =~ /env/
+		if tag['value'] == "prod"
+			subdomain = "#{region}.#{tag['value']}"
```
Comment (antgel) [inline lib/facter/ec2_tag_facts.rb]: I believe we agreed that all environments would have regions e.g. us-east-1.dev, us-east-1.qa, us-east-1.prod

File: lib/facter/ec2_tag_facts.rb [hunk 2]
```diff
@@ -105,6 +124,22 @@ def debug_msg(txt)
...
+		else
+			subdomain = "#{tag['value']}"
+		end
+		result["subdomain"] = subdomain
```
Comment (antgel) [inline lib/facter/ec2_tag_facts.rb]: I don't really like use of the term "subdomain", it's too vague. I thought we were going to keep `env` and `region` and then concatenate them. I don't mind having some concatenation, but I think it should be called region-env, not subdomain.

When the changes are made, please update https://github.com/graduway/psick/pull/87 and then I'll review that as well. Talk to me if anything isn't clear.

File: lib/facter/ec2_tag_facts.rb (hunk 1 above)
Comment (antgel) [reply lib/facter/ec2_tag_facts.rb]: @shapi78 This is still a thing. All environments should be able to handle a region.

**Comment by shapi78** [comment]
What about them? what to change?

File: lib/facter/ec2_tag_facts.rb (hunk 2 above)
Comment (shapi78) [reply lib/facter/ec2_tag_facts.rb]: changed to  result["region_env"]

**Pull request merged**